src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -2,73 +2,153 @@ import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
 import isEmpty from 'lodash/isEmpty';
 
+import '../sass/dashboard.scss';
+
 import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+
 import { focusElement } from '~/platform/utilities/ui';
+import {
+  isLOA3 as isLOA3Selector,
+  isLOA1 as isLOA1Selector,
+  isLoggedIn as isLoggedInSelector,
+} from '~/platform/user/selectors';
+
+import NameTag from '~/applications/personalization/profile/components/NameTag';
+import { fetchTotalDisabilityRating as fetchTotalDisabilityRatingAction } from '~/applications/personalization/rated-disabilities/actions';
 
 import {
   fetchMilitaryInformation as fetchMilitaryInformationAction,
   fetchHero as fetchHeroAction,
-  fetchPersonalInformation as fetchPersonalInformationAction,
 } from '@@profile/actions';
 
-import { fetchTotalDisabilityRating as fetchTotalDisabilityRatingAction } from '~/applications/personalization/rated-disabilities/actions';
-
-import NameTag from '~/applications/personalization/profile/components/NameTag';
-
+import IdentityNotVerified from './IdentityNotVerified';
 import ApplyForBenefits from './apply-for-benefits/ApplyForBenefits';
 import ClaimsAndAppeals from './claims-and-appeals/ClaimsAndAppeals';
 import HealthCare from './health-care/HealthCare';
 
-const Dashboard = props => {
-  useEffect(() => {
-    props.fetchFullName();
-    props.fetchPersonalInformation();
-    props.fetchMilitaryInformation();
-    props.fetchTotalDisabilityRating();
-    focusElement('#dashboard-title');
-  }, []);
-
-  return (
-    <>
-      {isEmpty(props.hero?.errors) && (
-        <NameTag
-          showUpdatedNameTag
-          totalDisabilityRating={props.totalDisabilityRating}
-        />
-      )}
-      <div className="vads-l-grid-container vads-u-padding-x--0">
-        <Breadcrumbs>
-          <a href="/" key="home">
-            Home
-          </a>
-          <span className="vads-u-color--black" key="dashboard">
-            <strong>My VA</strong>
-          </span>
-        </Breadcrumbs>
-
-        <h1 id="dashboard-title" data-testid="dashboard-title" tabIndex="-1">
-          My VA
-        </h1>
-
-        <ClaimsAndAppeals />
-        <HealthCare />
-        <ApplyForBenefits />
-      </div>
-    </>
+// content to show if the component is waiting for data to load. This loader
+// matches the loader shown by the RequiredLoginView component, so when the
+// RequiredLoginView is done with its loading and this component takes over, it
+// appears seamless to the user.
+const Loader = () => (
```

Comment:
Could we make this a shared component since the same Loader with the same margin is used on the profile? Not too opinionated, just a thought.

Reply:
Oh sure. I think it should live with the RequiredLoginView since it's based on the loader used there... that should be a different ticket, IMO.

Reply:
ticket for that: https://github.com/department-of-veterans-affairs/va.gov-team/issues/19511